<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Management</string>
    </property>
    <widget class="QPlainTextEdit" name="username">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Username : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Password : </string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="passowrd">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="register_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>240</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>387</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
